src/Presentation/RestSaveMappingsPresenter.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseRDF\Presentation;
+
+use MediaWiki\Rest\Response;
+use MediaWiki\Rest\ResponseFactory;
+use ProfessionalWiki\WikibaseRDF\Application\MappingList;
+use ProfessionalWiki\WikibaseRDF\Application\SaveMappings\SaveMappingsPresenter;
+use ProfessionalWiki\WikibaseRDF\MappingListSerializer;
+use Wikimedia\Message\MessageValue;
+
+class RestSaveMappingsPresenter implements SaveMappingsPresenter {
+
+	private Response $response;
+
+	public function __construct(
+		private ResponseFactory $responseFactory,
+		private MappingListSerializer $mappingListSerializer
+	) {
+	}
+
+	public function presentSuccess(): void {
+		$this->response = $this->responseFactory->createNoContent();
```

Comment:
I decided to return 204, since that indicates success with no content in the response. This is similar to MW's page update (https://www.mediawiki.org/wiki/API:REST_API/Reference#Responses_4) except they return the page object. Not sure if we really need to return anything except the status code.